src/app/components/AllowanceMenu/index.tsx

```diff
@@ -39,10 +42,18 @@ function AllowanceMenu({ allowance, onEdit, onDelete }: Props) {
   }
 
   async function updateAllowance() {
+    if (parseInt(budget) > allowance.remainingBudget) {
```

Comment:
This is an important part. It was `<` before I changed it. I believe this is correct.

Reply:
what is this check here? if the user tries to set a budget that is smaller than the remaining budget then we show an error? 

then it should be `<` ?